common/src/main/java/com/microsoft/identity/common/adal/internal/net/WebRequestHandler.java

```diff
@@ -94,4 +97,13 @@ public HttpWebResponse sendPost(URL url, Map<String, String> headers, byte[] con
     public void setRequestCorrelationId(final UUID requestCorrelationId) {
         mRequestCorrelationId = requestCorrelationId;
     }
+
+    @Override
+    public void setAdalClientVersion(String adalClientVersion) {
```

Comment:
Let's rename this to `setClientVersion` -- [MSAL uses this header as well](https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/f3861311f0ea1ee32cd30da7b548f8f701515775/msal/src/internal/java/com/microsoft/identity/client/PlatformIdHelper.java#L76) and ideally we will share all HTTP logic in the future.

Please update the parameter of this method to be `final clientVersion` -- don't forget to update the field name as well

Reply:
updated. thanks